pylib/spinnaker/codelab_config.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2016 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import spinnaker.yaml_util as yaml_util
+
+JENKINS_DEFAULT_MASTER_NAME = 'jenkins.defaultMaster.name'
+JENKINS_DEFAULT_MASTER_BASE = 'jenkins.defaultMaster.baseUrl'
+JENKINS_DEFAULT_MASTER_USERNAME = 'jenkins.defaultMaster.username'
+JENKINS_DEFAULT_MASTER_PASSWORD = 'jenkins.defaultMaster.password'
+IGOR_ENABLED = 'igor.enabled'
+PROP_KEYS = [
+    JENKINS_DEFAULT_MASTER_NAME,
+    JENKINS_DEFAULT_MASTER_BASE,
+    JENKINS_DEFAULT_MASTER_USERNAME,
+    JENKINS_DEFAULT_MASTER_PASSWORD
+]
+
+def configure_igor_jenkins():
+    """Configures Igor to be enabled and to point to the codelab jenkins instance.
+
+    """
+    yaml_binds = yaml_util.YamlBindings()
+    update_dict = {
+        'jenkins': {
+            'defaultMaster': {
+                'name': 'Jenkins',
```

Comment:
Since you are providing a jenkins server that has a particular configuration and installs for the tutorial, maybe you want to give this a more specific name, like CodelabJenkins?
Jenkins is fine too.